extensions/fluxninja/extconfig/extconfig.go

```diff
@@ -30,6 +33,8 @@ type FluxNinjaExtensionConfig struct {
 	InstallationMode string `json:"installation_mode" validate:"oneof=KUBERNETES_SIDECAR KUBERNETES_DAEMONSET LINUX_BARE_METAL" default:"LINUX_BARE_METAL"`
 	// Whether to configure local Prometheus OTel pipeline for metrics
 	DisableLocalOTelPipeline bool `json:"disable_local_otel_pipeline" default:"false"`
+	// Whether to enable cloud controller
```

Comment:
Please be more descriptive here (what connections are affected / overrided (etcd & tls, right)?